frontend/app/routes/$lang+/_public+/apply+/$id+/adult-child/applicant-information.tsx

```diff
@@ -221,7 +221,8 @@ export default function ApplyFlowApplicationInformation() {
               name="socialInsuranceNumber"
               label={t('applicant-information.sin')}
               inputMode="numeric"
-              placeholder="000-000-000"
+              helpMessagePrimary={t('apply-adult-child:applicant-information.error-message.sin-required')}
```

Comment:
Create a new locale for the SIN help message even though it's same content as the error message. One of them could different later.